modules/_labs/badge/react/stories/stories_visualTesting.tsx

```diff
@@ -0,0 +1,73 @@
+/// <reference path="../../../../../typings.d.ts" />
+import React from 'react';
+import withReadme from 'storybook-readme/with-readme';
+
+import {Table} from '../../../../../utils/storybook';
+
+import Badge from '../index';
+import README from '../README.md';
+
+export default {
+  title: 'Labs|Badge/React/',
+  decorators: [withReadme(README)],
+  parameters: {
+    chromatic: {
+      disable: false,
```

Comment:
@NicholasBoll to chime in when he figures out how to get this to work (chromatic is not catching these in the CSF format atm)

Reply:
ohhh. I was wondering if it needs to be:
```
  parameters: {
    options: {
      chromatic: {
        disable: false,
      },
  },
}
```

Like what's shown [here](https://storybook.js.org/docs/configurations/options-parameter/#per-story-options)

Reply:
It looks right, but it wasn't working for @NicholasBoll on #528. Will have to double check

Reply:
This looks good 👍  (see https://github.com/Workday/canvas-kit/pull/585)